ui-frontend/packages/catalog-ui-search/src/main/webapp/js/controllers/cesium.layerCollection.controller.js

```diff
@@ -42,6 +43,7 @@ const Controller = CommonLayerController.extend({
   initialize() {
     // there is no automatic chaining of initialize.
     CommonLayerController.prototype.initialize.apply(this, arguments)
+    this.listenTo(wreqr.vent, 'map:requestRender', this.requestRender)
```

Comment:
I think we should move this over to the initialization of the map in: https://github.com/codice/ddf-ui/blob/master/ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js#L103

We'll have to handle unlistening ourselves though there, so we'll have to update the destroy method as well: https://github.com/codice/ddf-ui/blob/master/ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/map.cesium.js#L1024